docs/basic-concepts.rst

```diff
@@ -1,64 +1,64 @@
 
-Runtimes provide the basic dependencies that are used by applications. Each
+Runtimes provide the basic dependencies used by applications. Each
 application must be built against a runtime, and this runtime must be
-installed on a host system in order for the application to run (Flatpak
-can automatically install the runtime required by an application). Multiple
-different runtimes can be installed at the same time, including different
-versions of the same runtime.
+installed on the host system for the application to run. (Flatpak
+can automatically install the required runtime for an application.) Multiple
+runtimes and different versions of the same runtime can be installed
+alongside each other.
 
-Runtimes are distribution agnostic and do not depend on particular distribution
+Runtimes are distribution-agnostic and do not depend on particular distribution
 versions. This means that they provide a stable, cross-distribution base
-for applications, and allow applications to continue to work irrespective
+for applications and allow applications to work irrespective
 of operating system updates.
 
 Bundled libraries
 -----------------
 
-If an application requires any dependencies that aren't in its runtime, they
-can be bundled as part of the application. This gives application developers
-flexibility regarding the dependencies that they use, including using:
+If an application requires dependencies that aren't in its runtime, they
+can be bundled with the application. This gives application developers
+flexibility in their choice of dependencies, allowing them to use:
 
 - libraries that aren't available in a runtime
-- different versions of libraries from the ones that are in a runtime
+- different versions of libraries from those available in a runtime
 - patched versions of libraries
 
 Sandboxes
 ---------
 
-With Flatpak, each application is built and run in an isolated environment,
-which is called the 'sandbox'. Each sandbox contains an application and
+With Flatpak, each application is built and run in an isolated environment
+called the 'sandbox'. Each sandbox contains the application and
 its runtime. By default, the application can only access the contents of
 its sandbox. Access to user files, network, graphics sockets, subsystems on
-the bus and devices have to be explicitly granted. Access to other things,
+the bus, and devices have to be explicitly granted. Access to other resources,
 such as other processes, is deliberately not possible.
 
-By necessity, some resources that are inside the sandbox need to be exposed
-outside, to be used by the host system. These are known as 'exports', since
-they are files that are exported out of the sandbox, and include things like
-the application's ``.desktop`` file and icon.
+By necessity, some resources inside the sandbox need to be exported
+outside to be used by the host system. These are known as 'exports'
+and include resources such as the application's ``.desktop`` file and
```

Comment:
```suggestion
and include resources such as the application's desktop file and
```

Reply:
I don't think this should be changed from the original with its formatting and the dot.

Reply:
No desktop file is correct and matchea the phrase used for icon. You don't say .png file or .png icon.